common/autoconf/toolchain_windows.m4

```diff
@@ -152,12 +152,6 @@ AC_DEFUN([TOOLCHAIN_CHECK_POSSIBLE_VISUAL_STUDIO_ROOT],
 
       if test "x$VS_ENV_CMD" = x; then
         AC_MSG_NOTICE([Warning: None of $VCVARSFILES were found, Visual Studio installation not recognized. Ignoring])
-      else
-        # PLATFORM_TOOLSET is used during the compilation of the freetype sources
-        # (see 'LIB_BUILD_FREETYPE' in libraries.m4) and must be one of 'v100',
-        # 'v110' or 'v120' for VS 2010, 2012 or VS2013
-        eval PLATFORM_TOOLSET="\${VS_VS_PLATFORM_NAME_${VS_VERSION}}"
-
```

Comment:
this change probably correct, but it is not part of the jdk11 [commit](https://github.com/openjdk/jdk/commit/58ff4ee8c4ec399528338b59eadea996110366a0#diff-098656ab8708ea86ee2cc19047997ed8f58caf22b2d239630276329fae933269), and this code still exists in the [openjdk/jdk](https://github.com/openjdk/jdk/blob/74066bcca82749722e6fee57469520d418bf3430/make/autoconf/toolchain_microsoft.m4#L105)

Reply:
good spot, I've pushed a commit to revert this and will see if a build still passes